tests/integration/components/polaris-drop-zone-test.js

```diff
@@ -241,17 +241,13 @@ module('Integration | Component | polaris-drop-zone', function(hooks) {
         assert.dom(fileUploadStackSelector).exists('fileUpload wraps content in a stack');
         assert.dom(fileUploadStackSelector).hasClass('Polaris-Stack--vertical', 'fileUpload stack is vertically aligned');
         assert.dom(fileUploadImageSelector).exists('fileUpload image is rendered');
-        assert.dom(fileUploadImageSelector).hasAttribute('src', '/ember-polaris/images/file-upload.svg', 'fileUpload has correct image `src` attribute');
```

Comment:
Can you replace these tests with tests that the correct SVG renders please? We have a mock `svg-jar` component which sets the source on the element's data set which we use e.g. [here](https://github.com/smile-io/ember-polaris/blob/master/tests/integration/components/polaris-action-list-test.js#L106)